README.md

```diff
@@ -111,10 +111,12 @@ import 'package:easy_localization/easy_localization.dart';
 void main(){
   runApp(EasyLocalization(
     child: MyApp(),
-    supportedLocales: [Locale('en', 'US'), Locale('ar', 'DZ')],
+    supportedLocales: [Locale('en', 'US'), Locale('ar', 'DZ')], // [Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant', countryCode: 'HK')]
```

Comment:
if possible found other way todo this without ` Locale.fromSubtags`

Reply:
nope... only use `fromSubtags`, locale conatain olnly two parameters   `this._languageCode this._countryCode`

https://api.flutter.dev/flutter/dart-ui/Locale/Locale.html

Reply:
Or create new Locale class and oweride... i think its bad way